ws-server/src/server.rs

```diff
@@ -179,18 +189,40 @@ async fn background_task(
 	}
 }
 
+#[derive(Debug, Clone)]
+enum Cors {
```

Comment:
Happy to change it. I think I named it `Cors` since that's what the analog enum is already called in substrate, but no reason for us to confuse things, plus abbreviations are confusing almost by definition.